tests/perf/CMakeLists.txt

```diff
@@ -88,3 +88,7 @@ seastar_add_test (allocator
 
 seastar_add_test (container
   SOURCES container_perf.cc)
+
+seastar_add_test (http_client
+ 0 SOURCES http_client_perf.cc
```

Comment:
what is this `0` for? might want to drop it.

Reply:
Weird indeed, will drop